spec/preservation_queue_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+require 'timecop'
+
+RSpec.describe PushmiPullyu::PreservationQueue do
```

Comment:
Should we do any unhappy path testing here? Or this can come later? Like in this ticket: https://github.com/ualbertalib/pushmi_pullyu/issues/27